### PR TITLE
Update training_and_evaluation.ipynb

### PR DESCRIPTION
**Pull request by kernelizd**
Just changing the line
`y_pred  = tf.argmax(y_pred, axis=1)
`
to

` y_pred = tf.reshape(tf.argmax(y_pred, axis=1), shape=(-1, 1))
`

Now categorical_true_postive works correctly and shows something close to 50k rather than 8k.

### CODE REVIEW

**Comment by kernelizd** [comment]
I signed it!

**Comment by kernelizd** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdocs%2Fpull%2F674) for more info**.

I signed it!

File: site/en/r2/guide/keras/training_and_evaluation.ipynb [hunk 1]
```diff
@@ -468,7 +420,7 @@
         "      self.true_positives = self.add_weight(name='tp', initializer='zeros')\n",
         "\n",
         "    def update_state(self, y_true, y_pred, sample_weight=None):\n",
-        "      y_pred = tf.argmax(y_pred)\n",
+        "      y_pred = tf.reshape(tf.argmax(y_pred, axis=1), shape=(-1, 1))\n",
```
Comment (lamberta) [inline site/en/r2/guide/keras/training_and_evaluation.ipynb]: @yashk2810 PTAL

**Comment by lamberta** [comment]
Thanks, @kernelizd 
Can you try signing the CLA again here:  https://cla.developers.google.com/ ? The bot is a bit confused.

File: site/en/r2/guide/keras/training_and_evaluation.ipynb (hunk 1 above)
Comment (yashk2810) [reply site/en/r2/guide/keras/training_and_evaluation.ipynb]: LGTM

**Comment by kernelizd** [comment]
Thanks for everything. I'm glad to have contributed.

**Comment by lamberta** [comment]
Thank you

**Pull request closed without merging**